modules/@apostrophecms/home-page/index.js

```diff
@@ -10,12 +12,7 @@ module.exports = {
         contextual: true,
         max: 1,
         options: {
-          widgets: {
-            '@apostrophecms/rich-text': {
-              toolbar: [],
-              styles: []
-            }
-          }
```

Comment:
I don't know why this area was configured to just have an empty rich text widget. Was it on purpose?